src/components/NavBar.js

```diff
@@ -1,26 +1,42 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { connect } from "react-redux";
 import logo from "../media/image/Logo_113x83.svg";
 import head from '../media/images/budget_blocks.png';
 import { Link } from "react-router-dom";
 import "./main.css";
-
 import { logoutUser } from "../redux/actions/LogoutAction";
-
+import { Redirect} from "react-router-dom";
 export function NavBar({ navState, logoutUser }) {
+  
+
+
+  const [initialClass, setClass] = useState(!!(sessionStorage.getItem('token')));
+
+  const changeClass  = () => {
+    sessionStorage.removeItem('token')
+    sessionStorage.removeItem('navState')
+    sessionStorage.removeItem('userID')
+    sessionStorage.removeItem('LinkedAccount')
+    setClass(false)
```

Comment:
Can we change this function name to be more semantic? It sounds like it would be affecting a style prop but it's actually affect sessionStorage clearance